### DISCUSSION TITLE
I can't submit form

### DISCUSSION THREAD

**Discussion by ToptenDev** [community/community#110817] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I'm working on a web development project, and I encountered a small issue. When I try to submit a form, it doesn't seem to be working properly. Can anyone help me figure out what's wrong?

`<form action="/submit-form" method="POST">
  <input type="text" name="username" placeholder="Username">
  <input type="submit" value="Submit">
</form>`

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by iamsuiux** [comment, marked as answer]
I geuss you forgot to add the name attribute to the submit button.
Here's the corrected code:
`<form action="/submit-form" method="POST">
  <input type="text" name="username" placeholder="Username">
  <input type="submit" name="submit" value="Submit">
</form>`

**Comment by queenofcorgis** [comment]
Hey there - it does not look like this particular discussion has to with Issues, Projects, or GitHub. Make sure you're including all relevant details in your posts as off topic posts will be removed. 

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
